geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -155,7 +155,11 @@ Other Docker images
 Main Changes
 ------------
 
-The migration process will do the following UI-related changes (visible in the html/ejs diffs):
+The migration process impacts the following:
+1. Openlayers view has a default value to `false` for the `constrainOnlyCenter` parameter.
```

Comment:
default value to `false` 
-> default value of `false`
or
-> default value set to `false`